app/app_test.go

```diff
@@ -734,7 +734,6 @@ func TestGetRecorder(t *testing.T) {
 			ctx := context.Background()
 			r := app.GetRecorder(ctx, sessionId)
 			assert.NotNil(t, r)
-			assert.Equal(t, store, r.store)
```

Comment:
my removed? follow up question to self why was it here.

Reply:
so, recorders became officially `io.Writer`s in `app` function signatures.
this is for easier mocking (it's super simple to write your own mock writer or use `ioutil.Discard`)